versioned_docs/version-1.19/release-notes.mdx

```diff
@@ -5,6 +5,16 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.19.2
```

Comment:
You need to update his file too:
https://github.com/okteto/docs/blob/main/src/content/release-notes.mdx